### DISCUSSION TITLE
[A-L] (2023/24) Foglio 3 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#188] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-10-22 204529](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/ea4c7aa6-015b-43b1-a929-39b237e18191)

**Comment by Elia-Belli** [comment, marked as answer]
$\mathbb{Z}\_{24} =\\{[0],[1],[2],...,[23]\\}$
$\mathcal{U}(\mathbb{Z}\_{24} ):=\\{[a] \in \mathbb{Z}\_{24} | (a,24)=1\\}=\\{[1],[5],[7],[11],[13],[17],[19],[23]\\}$
$A=\\{[a]\in \mathcal{U}(\mathbb{Z}\_{24})| [a][a]=[1]\\}=\\{[1],[5],[7],[11],[13],[17],[19],[23]\\}$ 
($A= \mathcal{U}(\mathbb{Z}\_{24})$ per puro caso)

**Comment by daromacarlo** [comment]
![Immagine 2023-12-12 131828](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/2d81bdcd-1cde-4936-9aba-8050e6dee01d)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/c088711a-9c4d-4bee-91ba-c8195a7ad3da)
